portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,17 +31,21 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
     synchronized(comments){
-       comments.add("Hello World!");
-       comments.add("Hola Mundo!");
-       comments.add("Hallo Welt!");
        response.setContentType("application/json;");
        String json = convertToJsonUsingGson(comments);
        response.getWriter().println(json); 
-       comments.clear();
     }
     
-    
   }
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      //get input from the form
```

Comment:
Keep comments consistent. Capitalize the first letter and end with a period (e.g. // Get input from the form.)